conf/mccleary.config

```diff
@@ -0,0 +1,32 @@
+//Profile config names for nf-core/configs
+params {
+    config_profile_description = 'McCleary Cluster at Yale'
+    config_profile_contact = 'Gisela Gabernet'
```

Comment:
Name is enough to find you there?

Reply:
good point, just added the full contact info, thanks!